packages/asset/contracts/libraries/TokenIdUtils.sol

```diff
@@ -99,11 +92,11 @@ library TokenIdUtils {
     }
 
     /// @notice Extracts the abilities and enhancements hash from a given token id
-    /// @param tokenId The token id to extract the abilities and enhancements hash from
-    /// @return revealHash The reveal hash of the abilities and enhancements of the asset
-    function getRevealHash(uint256 tokenId) internal pure returns (uint40) {
-        uint40 ipfsHash = uint40((tokenId >> HASH_SHIFT) & HASH_MASK);
-        return ipfsHash;
+    /// @param tokenId The token id to extract reveal nonce from
+    /// @return revealNonce The reveal nonce of the asset
+    function getRevealNonce(uint256 tokenId) internal pure returns (uint16) {
+        uint16 revealNonce = uint16((tokenId >> REVEAL_NONCE_SHIFT) & REVEAL_NONCE_MASK);
+        return revealNonce;
```

Comment:
Do we even need this? Do you think we will use this?

Reply:
We could use it to get the reveal version of asset like we discussed

Reply:
I think we use it mostly to know if an asset has been revealed or not, ok keeping it.